node_test/helpers_test.go

```diff
@@ -55,6 +55,7 @@ func waitForPeerInfoExchange(services ...*p2pms.P2PMessageService) {
 			peerInfo := <-s.PeerInfoReceived()
 			fmt.Printf("Service num: %d, peer num: %d, peerInfo: %v\n", sNum, i, peerInfo)
 		}
+		<-s.InitComplete
```

Comment:
I wonder if this should be encapsulated into a method on `s`? e.g. `WaitForInitComplete`.

Do we anticipate using this in production (and not just test) code?

Reply:
I guess it's not a huge difference. Advantages are that it is the way the chan is being used is made more explicit, and that it may be more maintainable (in case e.g.  we end up needing to wait on anything else). 

Your call if it is worth it, here.

Reply:
Ha, I made a similar [comment](https://github.com/statechannels/go-nitro/pull/1515/files#r1293895933) before noticing this one. 

Another advantage of using a function is that it let's us specify the direction of the channel, which adds some guardrails. Right now anyone can read or **send** to `s.InitComplete`. If we used a function to return a one way chan, we ensure that only the message service can send on the chan.